### ISSUE TITLE
Transfer data to new phone

### ISSUE THREAD

**Issue by xtough** [boolder-org/boolder-android#168]
Hi all,
first of all: great app. We found this indispensable when navigating unknown terrain in Bleau. 👍 

When switching phones recently, I noticed that my tick list had vanished from the Boolder app on the new phone, despite the fact that I used the Android data transfer feature from old to new phone. When looking manually for the data folder I could not find anything on the old phone under /Android/data/com.boolder*

I know from #115 that an account transfer feature has been in discussion. But is there already today a possibility to get my tick list from one phone to another? 

Thx 💯 in advance!
Christoph

**Comment by wang-li** [comment]
Hi @xtough 👋 

As you mentioned, the accounts are not yet implemented in the Boolder app, but an export / import feature is available from the tick list tab, accessible from the three dots button in the upper right corner of the screen:
<img src="https://github.com/user-attachments/assets/6d2823e9-cbcd-4f65-a9de-79d77324b7be" width=400 />

For instance, you can export to Google Drive on your old device, and import from the same Google Drive folder on your new device.

**Comment by maxgabut** [comment]
I just used the Android's built in data transfer feature, and my tick list was transferred. Maybe this was fixed and the issue can be closed.

**Comment by xtough** [comment]
> I just used the Android's built in data transfer feature, and my tick list was transferred. Maybe this was fixed and the issue can be closed.

I agree. The original request is fulfilled by the export/import feature. I was also able to use it successfully. Thanks @wang-li 

Online accounts is yet another story / feature request. I would recommend to handle this separately, because it has a much broader impact.
